pom.xml

```diff
@@ -76,9 +80,14 @@
     <dependency>
       <groupId>org.testcontainers</groupId>
       <artifactId>postgresql</artifactId>
-      <version>1.15.2</version>
+      <version>${pgcontainer.version}</version>
```

Comment:
```suggestion
      <version>${testcontainers.version}</version>
```

Reply:
Yes, I tried it and works well, but I wasn't sure if you are using that posgresql container version for some specific reason if not, then yes let's use the same as testcontainer version.